### PR TITLE
[vcpkg] Added linux release triplets for arm, arm64, ppc, and s390x.

### PR DESCRIPTION
**Pull request by sarmadka**
Added release version of the following community triplets:
arm-linux
arm64-linux
ppc64le-linux
s390x-linux

- #### What does your PR fix?
Allows building on ARM, PowerPC, and s390x Linux platforms without needing to build unneeded debug version, which makes the build take longer unnecessarily and also in my case was failing in snapcraft.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
Added the following new triplets:
arm-linux-release
arm64-linux-release
ppc64le-linux-release
s390x-linux-release
These triplets are copies of the non-release counterparts, but adding the following line to them:
`set(VCPKG_BUILD_TYPE release)`

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
No, this change is pretty trivial so didn't spend time reading through the guide.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
No port added.

### CODE REVIEW

**Comment by sarmadka** [comment]
@microsoft-github-policy-service agree [company="Alusus Software Ltd."]

**Review by JonLiu1993 — DISMISSED** [review]

File: triplets/community/arm-linux-release.cmake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+set(VCPKG_TARGET_ARCHITECTURE arm)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+
+set(VCPKG_CMAKE_SYSTEM_NAME Linux)
+set(VCPKG_BUILD_TYPE release)
+
+if(NOT CMAKE_HOST_SYSTEM_PROCESSOR)
+    execute_process(COMMAND "uname" "-m" OUTPUT_VARIABLE CMAKE_HOST_SYSTEM_PROCESSOR OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif()
```
Comment (autoantwort) [inline triplets/community/arm-linux-release.cmake:10]: Is that really necessary?

**Review by vicroms — CHANGES_REQUESTED** [review]

File: triplets/community/arm-linux-release.cmake (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarmadka) [reply triplets/community/arm-linux-release.cmake:10]: I'm not really sure why this was needed to be honest; I just copied it from the non-release triplets. I can remove it from both if you think that's the right thing to do.

**Comment by daschuer** [comment]
A more straight forward approach would be implementing this: 
https://github.com/microsoft/vcpkg/issues/10683

It will be a probably a waste of resources if you like to build a Debug build later and it lands than in a different triplet sub-folder. 
This means you need to handle two complete install folders, instead of just adding or removing the debug sub-folders.  
It is also an issue to keep the release prefixed version in sync with the normal version. 
This is already an issue here, because the diff does not reveal the code lines you have actually added.

File: triplets/community/arm-linux-release.cmake (hunk 1 above)
[the remark this answers is not in the document]
Comment (vicroms) [reply triplets/community/arm-linux-release.cmake:10]: Please do

**Review by vicroms — CHANGES_REQUESTED** [review]
Remove setting of `CMAKE_HOST_SYSTEM_PROCESSOR` on triplets

File: triplets/community/arm-linux-release.cmake (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarmadka) [reply triplets/community/arm-linux-release.cmake]: done

**Review by vicroms — APPROVED** [review]

**Pull request merged**